src/po/sr.po

```diff
@@ -5027,18 +5010,74 @@ msgstr "E388: Дефиниција не може да се пронађе"
 msgid "E389: Couldn't find pattern"
 msgstr "E389: Шаблон за претрагу није пронађен"
 
-msgid "Substitute "
-msgstr "Замена "
+msgid "Save View"
+msgstr "Сачувај Поглед"
```

Comment:
"Pogled" in this context sounds silly, but I admit I can't find the right word for essentially "layout".